<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newOrderButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Новый заказ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QLabel" name="amountLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во товаров:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="resultSumLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Итого:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="resultSumLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="checkPreviewText">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>311</width>
      <height>431</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="formButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сформировать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dbTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>561</width>
      <height>431</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Скидка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Количество</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="searchLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск по названию:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLine">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFilePathButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать путь сохранения</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
